opentech/settings/test.py

```diff
@@ -8,6 +8,8 @@
 
 SECRET_KEY = 'NOT A SECRET'
 
+PROJECTS_ENABLED = True
```

Comment:
You shouldn't need this, the Travis and Circle configs both add it as an environment variable.